src-docs/src/views/tables/sorting/sorting_section.js

```diff
@@ -24,7 +24,11 @@ export const section = {
       The following example shows how to configure column sorting via the{' '}
       <EuiCode>sorting</EuiCode>
       property and flagging the sortable columns as{' '}
-      <EuiCode>sortable: true</EuiCode>
+      <EuiCode>sortable: true</EuiCode>.To enable the default sorting ability
```

Comment:
One last formatting issue that I can see

```suggestion
      <EuiCode>sortable: true</EuiCode>. To enable the default sorting ability
```